app/uk/gov/hmrc/breathingspaceifstub/controller/DebtorController.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.breathingspaceifstub.controller
+
+import javax.inject.{Inject, Singleton}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+import play.api.http.Status.{NOT_FOUND, OK, UNPROCESSABLE_ENTITY}
+import play.api.libs.json.JsValue
+import play.api.mvc._
+import uk.gov.hmrc.breathingspaceifstub.controller.DebtorController.sendResponse
+import uk.gov.hmrc.breathingspaceifstub.utils.ControllerSupport
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+@Singleton()
+class DebtorController @Inject()(
```

Comment:
What about `IndividualDetailsController` ?

Reply:
If you're ok with it I prefer 'debtor' as it's name as that is the entity this controller is concerned with. Get identity details is just one of the endpoints this controller could service for a specific debtor (if the need ever arose for adding more). WDYT?

Reply:
OK, np.